Comparing source compatibility of /home/jenkins/.m2/repository/org/apache/jspwiki/jspwiki-util/2.11.3-SNAPSHOT/jspwiki-util-2.11.3-SNAPSHOT.jar against /home/jenkins/.m2/repository/org/apache/jspwiki/jspwiki-util/2.11.0/jspwiki-util-2.11.0.jar
WARNING: You are using the option '--ignore-missing-classes', i.e. superclasses and interfaces that could not be found on the classpath are ignored. Hence changes caused by these superclasses and interfaces are not reflected in the output.
===  UNCHANGED CLASS: PUBLIC org.apache.wiki.util.ByteUtils  (not serializable)
	===  CLASS FILE FORMAT VERSION: 52.0 <- 52.0
	===  UNCHANGED SUPERCLASS: java.lang.Object (<- java.lang.Object)
	===  UNCHANGED METHOD: PUBLIC STATIC java.lang.String byte2hex(byte)
	===  UNCHANGED METHOD: PUBLIC STATIC java.lang.String bytes2hex(byte[])
	===  UNCHANGED METHOD: PUBLIC STATIC int hexToBin(char)
	===  UNCHANGED METHOD: PUBLIC STATIC byte[] parseHexBinary(java.lang.String)
+++  NEW INTERFACE: PUBLIC(+) ABSTRACT(+) org.apache.wiki.util.CheckedSupplier  (not serializable)
	+++  CLASS FILE FORMAT VERSION: 52.0 <- n.a.
	+++  NEW SUPERCLASS: java.lang.Object
	+++  NEW METHOD: PUBLIC(+) ABSTRACT(+) java.lang.Object get()
		+++  NEW EXCEPTION: java.lang.Exception
	+++  NEW ANNOTATION: java.lang.FunctionalInterface
***  MODIFIED CLASS: PUBLIC FINAL org.apache.wiki.util.ClassUtil  (not serializable)
	===  CLASS FILE FORMAT VERSION: 52.0 <- 52.0
	===  UNCHANGED SUPERCLASS: java.lang.Object (<- java.lang.Object)
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL java.lang.String MAPPINGS
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL java.lang.String MAPPINGS_EXTRA
	===  UNCHANGED METHOD: PUBLIC STATIC boolean assignable(java.lang.String, java.lang.String)
	+++  NEW METHOD: PUBLIC(+) STATIC(+) java.lang.Object buildInstance(java.lang.String)
		+++  NEW EXCEPTION: java.lang.ReflectiveOperationException
	+++  NEW METHOD: PUBLIC(+) STATIC(+) java.lang.Object buildInstance(java.lang.String, java.lang.String)
		+++  NEW EXCEPTION: java.lang.ReflectiveOperationException
	+++  NEW METHOD: PUBLIC(+) STATIC(+) java.lang.Object buildInstance(java.lang.Class)
		+++  NEW EXCEPTION: java.lang.ReflectiveOperationException
	+++  NEW METHOD: PUBLIC(+) STATIC(+) java.lang.Object buildInstance(java.lang.Class, java.lang.Object[])
		+++  NEW EXCEPTION: java.lang.ReflectiveOperationException
	===  UNCHANGED METHOD: PUBLIC STATIC java.util.List classpathEntriesUnder(java.lang.String)
	===  UNCHANGED METHOD: PUBLIC STATIC boolean exists(java.lang.String)
	===  UNCHANGED METHOD: PUBLIC STATIC java.lang.Class findClass(java.util.List, java.util.List, java.lang.String)
		===  UNCHANGED EXCEPTION: java.lang.ClassNotFoundException
	===  UNCHANGED METHOD: PUBLIC STATIC java.lang.Class findClass(java.lang.String, java.lang.String)
		===  UNCHANGED EXCEPTION: java.lang.ClassNotFoundException
	===  UNCHANGED METHOD: PUBLIC STATIC java.util.Map getExtraClassMappings()
	===  UNCHANGED METHOD: PUBLIC STATIC java.lang.Class getMappedClass(java.lang.String)
		===  UNCHANGED EXCEPTION: java.lang.ClassNotFoundException
	===  UNCHANGED METHOD: PUBLIC STATIC java.lang.Object getMappedObject(java.lang.String)
		===  UNCHANGED EXCEPTION: java.lang.ReflectiveOperationException
		===  UNCHANGED EXCEPTION: java.lang.IllegalArgumentException
	===  UNCHANGED METHOD: PUBLIC STATIC java.lang.Object getMappedObject(java.lang.String, java.lang.Object[])
		===  UNCHANGED EXCEPTION: java.lang.ReflectiveOperationException
		===  UNCHANGED EXCEPTION: java.lang.IllegalArgumentException
===  UNCHANGED CLASS: PUBLIC org.apache.wiki.util.CommentedProperties  (compatible)
	===  CLASS FILE FORMAT VERSION: 52.0 <- 52.0
	===  UNCHANGED INTERFACE: java.lang.Cloneable
	===  UNCHANGED INTERFACE: java.util.Map
	===  UNCHANGED INTERFACE: java.io.Serializable
	===  UNCHANGED SUPERCLASS: java.util.Properties (<- java.util.Properties)
	===  UNCHANGED CONSTRUCTOR: PUBLIC CommentedProperties()
	===  UNCHANGED CONSTRUCTOR: PUBLIC CommentedProperties(java.util.Properties)
	===  UNCHANGED METHOD: PUBLIC void load(java.io.InputStream)
		===  UNCHANGED EXCEPTION: java.io.IOException
	===  UNCHANGED METHOD: PUBLIC void load(java.io.Reader)
		===  UNCHANGED EXCEPTION: java.io.IOException
	===  UNCHANGED METHOD: PUBLIC java.lang.Object put(java.lang.Object, java.lang.Object)
	===  UNCHANGED METHOD: PUBLIC void putAll(java.util.Map)
	===  UNCHANGED METHOD: PUBLIC java.lang.Object remove(java.lang.Object)
	===  UNCHANGED METHOD: PUBLIC java.lang.Object setProperty(java.lang.String, java.lang.String)
	===  UNCHANGED METHOD: PUBLIC void store(java.io.OutputStream, java.lang.String)
		===  UNCHANGED EXCEPTION: java.io.IOException
	===  UNCHANGED METHOD: PUBLIC java.lang.String toString()
===  UNCHANGED CLASS: PUBLIC org.apache.wiki.util.comparators.CollatorComparator  (not serializable)
	===  CLASS FILE FORMAT VERSION: 52.0 <- 52.0
	===  UNCHANGED INTERFACE: java.util.Comparator
	===  UNCHANGED SUPERCLASS: java.lang.Object (<- java.lang.Object)
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL java.util.Comparator DEFAULT_LOCALE_COMPARATOR
	===  UNCHANGED FIELD: PROTECTED java.text.Collator m_collator
	===  UNCHANGED CONSTRUCTOR: PUBLIC CollatorComparator(java.text.Collator)
	===  UNCHANGED CONSTRUCTOR: PUBLIC CollatorComparator()
	===  UNCHANGED METHOD: PUBLIC int compare(java.lang.String, java.lang.String)
	===  UNCHANGED METHOD: PUBLIC void setCollator(java.text.Collator)
===  UNCHANGED CLASS: PUBLIC org.apache.wiki.util.comparators.HumanComparator  (not serializable)
	===  CLASS FILE FORMAT VERSION: 52.0 <- 52.0
	===  UNCHANGED INTERFACE: java.util.Comparator
	===  UNCHANGED SUPERCLASS: java.lang.Object (<- java.lang.Object)
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL java.util.Comparator DEFAULT_HUMAN_COMPARATOR
	===  UNCHANGED CONSTRUCTOR: PUBLIC HumanComparator(org.apache.wiki.util.comparators.HumanComparator$CharType[])
	===  UNCHANGED CONSTRUCTOR: PUBLIC HumanComparator()
	===  UNCHANGED METHOD: PUBLIC int compare(java.lang.String, java.lang.String)
	===  UNCHANGED METHOD: PUBLIC STATIC java.util.Comparator getInstance()
	===  UNCHANGED METHOD: PUBLIC org.apache.wiki.util.comparators.HumanComparator$CharType[] getSortOrder()
	===  UNCHANGED METHOD: PUBLIC void setSortOrder(org.apache.wiki.util.comparators.HumanComparator$CharType[])
===  UNCHANGED ENUM: PUBLIC STATIC FINAL org.apache.wiki.util.comparators.HumanComparator$CharType  (compatible)
	===  CLASS FILE FORMAT VERSION: 52.0 <- 52.0
	===  UNCHANGED INTERFACE: java.lang.Comparable
	===  UNCHANGED INTERFACE: java.io.Serializable
	===  UNCHANGED SUPERCLASS: java.lang.Enum (<- java.lang.Enum)
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL org.apache.wiki.util.comparators.HumanComparator$CharType TYPE_LETTER
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL org.apache.wiki.util.comparators.HumanComparator$CharType TYPE_DIGIT
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL org.apache.wiki.util.comparators.HumanComparator$CharType TYPE_OTHER
	===  UNCHANGED METHOD: PUBLIC STATIC org.apache.wiki.util.comparators.HumanComparator$CharType valueOf(java.lang.String)
	===  UNCHANGED METHOD: PUBLIC STATIC org.apache.wiki.util.comparators.HumanComparator$CharType[] values()
===  UNCHANGED CLASS: PUBLIC org.apache.wiki.util.comparators.JavaNaturalComparator  (not serializable)
	===  CLASS FILE FORMAT VERSION: 52.0 <- 52.0
	===  UNCHANGED INTERFACE: java.util.Comparator
	===  UNCHANGED SUPERCLASS: java.lang.Object (<- java.lang.Object)
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL java.util.Comparator DEFAULT_JAVA_COMPARATOR
	===  UNCHANGED CONSTRUCTOR: PUBLIC JavaNaturalComparator()
	===  UNCHANGED METHOD: PUBLIC int compare(java.lang.String, java.lang.String)
===  UNCHANGED CLASS: PUBLIC org.apache.wiki.util.comparators.LocaleComparator  (not serializable)
	===  CLASS FILE FORMAT VERSION: 52.0 <- 52.0
	===  UNCHANGED INTERFACE: java.util.Comparator
	===  UNCHANGED SUPERCLASS: org.apache.wiki.util.comparators.CollatorComparator (<- org.apache.wiki.util.comparators.CollatorComparator)
	===  UNCHANGED CONSTRUCTOR: PUBLIC LocaleComparator()
	===  UNCHANGED CONSTRUCTOR: PUBLIC LocaleComparator(java.util.Locale)
	===  UNCHANGED METHOD: PUBLIC void setLocale(java.util.Locale)
===  UNCHANGED CLASS: PUBLIC org.apache.wiki.util.comparators.PrincipalComparator  (compatible)
	===  CLASS FILE FORMAT VERSION: 52.0 <- 52.0
	===  UNCHANGED INTERFACE: java.util.Comparator
	===  UNCHANGED INTERFACE: java.io.Serializable
	===  UNCHANGED SUPERCLASS: java.lang.Object (<- java.lang.Object)
	===  UNCHANGED CONSTRUCTOR: PUBLIC PrincipalComparator()
	===  UNCHANGED METHOD: PUBLIC int compare(java.security.Principal, java.security.Principal)
===  UNCHANGED CLASS: PUBLIC FINAL org.apache.wiki.util.CryptoUtil  (not serializable)
	===  CLASS FILE FORMAT VERSION: 52.0 <- 52.0
	===  UNCHANGED SUPERCLASS: java.lang.Object (<- java.lang.Object)
	===  UNCHANGED METHOD: PUBLIC STATIC java.lang.String getSaltedPassword(byte[], java.lang.String)
		===  UNCHANGED EXCEPTION: java.security.NoSuchAlgorithmException
	===  UNCHANGED METHOD: PUBLIC STATIC void main(java.lang.String[])
		===  UNCHANGED EXCEPTION: java.lang.Exception
	===  UNCHANGED METHOD: PUBLIC STATIC boolean verifySaltedPassword(byte[], java.lang.String)
		===  UNCHANGED EXCEPTION: java.security.NoSuchAlgorithmException
===  UNCHANGED CLASS: PUBLIC FINAL org.apache.wiki.util.FileUtil  (not serializable)
	===  CLASS FILE FORMAT VERSION: 52.0 <- 52.0
	===  UNCHANGED SUPERCLASS: java.lang.Object (<- java.lang.Object)
	===  UNCHANGED METHOD: PUBLIC STATIC void copyContents(java.io.Reader, java.io.Writer)
		===  UNCHANGED EXCEPTION: java.io.IOException
	===  UNCHANGED METHOD: PUBLIC STATIC void copyContents(java.io.InputStream, java.io.OutputStream)
		===  UNCHANGED EXCEPTION: java.io.IOException
	===  UNCHANGED METHOD: PUBLIC STATIC java.lang.String getThrowingMethod(java.lang.Throwable)
	===  UNCHANGED METHOD: PUBLIC STATIC java.io.File newTmpFile(java.lang.String, java.nio.charset.Charset)
		===  UNCHANGED EXCEPTION: java.io.IOException
	===  UNCHANGED METHOD: PUBLIC STATIC java.io.File newTmpFile(java.lang.String)
		===  UNCHANGED EXCEPTION: java.io.IOException
	===  UNCHANGED METHOD: PUBLIC STATIC java.lang.String readContents(java.io.InputStream, java.lang.String)
		===  UNCHANGED EXCEPTION: java.io.IOException
	===  UNCHANGED METHOD: PUBLIC STATIC java.lang.String readContents(java.io.Reader)
		===  UNCHANGED EXCEPTION: java.io.IOException
	===  UNCHANGED METHOD: PUBLIC STATIC java.lang.String runSimpleCommand(java.lang.String, java.lang.String)
		===  UNCHANGED EXCEPTION: java.io.IOException
		===  UNCHANGED EXCEPTION: java.lang.InterruptedException
===  UNCHANGED CLASS: PUBLIC FINAL org.apache.wiki.util.FormUtil  (not serializable)
	===  CLASS FILE FORMAT VERSION: 52.0 <- 52.0
	===  UNCHANGED SUPERCLASS: java.lang.Object (<- java.lang.Object)
	===  UNCHANGED METHOD: PUBLIC STATIC java.util.List getNumberedValues(java.util.Map, java.lang.String)
	===  UNCHANGED METHOD: PUBLIC STATIC java.util.List getValues(java.util.Map, java.lang.String)
	===  UNCHANGED METHOD: PUBLIC STATIC java.util.Map requestToMap(javax.servlet.http.HttpServletRequest, java.lang.String)
===  UNCHANGED CLASS: PUBLIC FINAL org.apache.wiki.util.HttpUtil  (not serializable)
	===  CLASS FILE FORMAT VERSION: 52.0 <- 52.0
	===  UNCHANGED SUPERCLASS: java.lang.Object (<- java.lang.Object)
	===  UNCHANGED METHOD: PUBLIC STATIC boolean checkFor304(javax.servlet.http.HttpServletRequest, java.lang.String, java.util.Date)
	===  UNCHANGED METHOD: PUBLIC STATIC void clearCookie(javax.servlet.http.HttpServletResponse, java.lang.String)
	===  UNCHANGED METHOD: PUBLIC STATIC java.lang.String createETag(java.lang.String, java.util.Date)
	===  UNCHANGED METHOD: PUBLIC STATIC java.lang.String getRemoteAddress(javax.servlet.http.HttpServletRequest)
	===  UNCHANGED METHOD: PUBLIC STATIC java.lang.String guessValidURI(java.lang.String)
	===  UNCHANGED METHOD: PUBLIC STATIC boolean isIPV4Address(java.lang.String)
	===  UNCHANGED METHOD: PUBLIC STATIC java.lang.String retrieveCookieValue(javax.servlet.http.HttpServletRequest, java.lang.String)
	===  UNCHANGED METHOD: PUBLIC STATIC java.lang.String safeGetQueryString(javax.servlet.http.HttpServletRequest, java.nio.charset.Charset)
===  UNCHANGED CLASS: PUBLIC FINAL org.apache.wiki.util.MailUtil  (not serializable)
	===  CLASS FILE FORMAT VERSION: 52.0 <- 52.0
	===  UNCHANGED SUPERCLASS: java.lang.Object (<- java.lang.Object)
	===  UNCHANGED METHOD: PUBLIC STATIC void sendMessage(java.util.Properties, java.lang.String, java.lang.String, java.lang.String)
		===  UNCHANGED EXCEPTION: javax.mail.internet.AddressException
		===  UNCHANGED EXCEPTION: javax.mail.MessagingException
===  UNCHANGED CLASS: PROTECTED STATIC org.apache.wiki.util.MailUtil$SmtpAuthenticator  (not serializable)
	===  CLASS FILE FORMAT VERSION: 52.0 <- 52.0
	===  UNCHANGED SUPERCLASS: n.a.
	===  UNCHANGED CONSTRUCTOR: PUBLIC MailUtil$SmtpAuthenticator(java.lang.String, java.lang.String)
	===  UNCHANGED METHOD: PUBLIC javax.mail.PasswordAuthentication getPasswordAuthentication()
===  UNCHANGED CLASS: PUBLIC org.apache.wiki.util.PriorityList  (not serializable)
	===  CLASS FILE FORMAT VERSION: 52.0 <- 52.0
	===  UNCHANGED INTERFACE: java.util.List
	===  UNCHANGED INTERFACE: java.util.Collection
	===  UNCHANGED INTERFACE: java.lang.Iterable
	===  UNCHANGED SUPERCLASS: java.util.AbstractList (<- java.util.AbstractList)
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL int DEFAULT_PRIORITY
	===  UNCHANGED CONSTRUCTOR: PUBLIC PriorityList()
	===  UNCHANGED METHOD: PUBLIC void add(java.lang.Object, int)
	===  UNCHANGED METHOD: PUBLIC boolean add(java.lang.Object)
	===  UNCHANGED METHOD: PUBLIC java.lang.Object get(int)
	===  UNCHANGED METHOD: PUBLIC int size()
===  UNCHANGED CLASS: PUBLIC FINAL org.apache.wiki.util.PropertiesUtils  (not serializable)
	===  CLASS FILE FORMAT VERSION: 52.0 <- 52.0
	===  UNCHANGED SUPERCLASS: java.lang.Object (<- java.lang.Object)
	===  UNCHANGED METHOD: PUBLIC STATIC java.lang.String saveConvert(java.lang.String, boolean)
	===  UNCHANGED METHOD: PUBLIC STATIC java.lang.String toLine(java.lang.String, java.lang.String)
	===  UNCHANGED METHOD: PUBLIC STATIC java.lang.String toSortedString(java.util.Properties)
===  UNCHANGED CLASS: PUBLIC FINAL org.apache.wiki.util.PropertyReader  (not serializable)
	===  CLASS FILE FORMAT VERSION: 52.0 <- 52.0
	===  UNCHANGED SUPERCLASS: java.lang.Object (<- java.lang.Object)
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL java.lang.String PARAM_CUSTOMCONFIG
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL java.lang.String CUSTOM_JSPWIKI_CONFIG
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL java.lang.String DEFAULT_JSPWIKI_CONFIG
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL java.lang.String PARAM_CUSTOMCONFIG_CASCADEPREFIX
	===  UNCHANGED METHOD: PUBLIC STATIC void expandVars(java.util.Properties)
	===  UNCHANGED METHOD: PUBLIC STATIC java.util.Properties getCombinedProperties(java.lang.String)
	===  UNCHANGED METHOD: PUBLIC STATIC java.util.Properties getDefaultProperties()
	===  UNCHANGED METHOD: PUBLIC STATIC java.util.Properties loadWebAppProps(javax.servlet.ServletContext)
	===  UNCHANGED METHOD: PUBLIC STATIC java.io.InputStream locateClassPathResource(javax.servlet.ServletContext, java.lang.String)
===  UNCHANGED CLASS: PUBLIC FINAL org.apache.wiki.util.Serializer  (not serializable)
	===  CLASS FILE FORMAT VERSION: 52.0 <- 52.0
	===  UNCHANGED SUPERCLASS: java.lang.Object (<- java.lang.Object)
	===  UNCHANGED METHOD: PUBLIC STATIC java.util.Map deserializeFromBase64(java.lang.String)
		===  UNCHANGED EXCEPTION: java.io.IOException
	===  UNCHANGED METHOD: PUBLIC STATIC java.lang.String serializeToBase64(java.util.Map)
		===  UNCHANGED EXCEPTION: java.io.IOException
***  MODIFIED CLASS: PUBLIC FINAL org.apache.wiki.util.TextUtil  (not serializable)
	===  CLASS FILE FORMAT VERSION: 52.0 <- 52.0
	===  UNCHANGED SUPERCLASS: java.lang.Object (<- java.lang.Object)
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL java.lang.String LEGACY_CHARS_ALLOWED
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL java.lang.String PUNCTUATION_CHARS_ALLOWED
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL int PASSWORD_LENGTH
	===  UNCHANGED METHOD: PUBLIC STATIC java.lang.String beautifyString(java.lang.String)
	===  UNCHANGED METHOD: PUBLIC STATIC java.lang.String beautifyString(java.lang.String, java.lang.String)
	===  UNCHANGED METHOD: PUBLIC STATIC java.lang.String cleanString(java.lang.String, java.lang.String)
	===  UNCHANGED METHOD: PUBLIC STATIC int countSections(java.lang.String)
	===  UNCHANGED METHOD: PUBLIC STATIC java.util.Properties createProperties(java.lang.String[])
		===  UNCHANGED EXCEPTION: java.lang.IllegalArgumentException
	===  UNCHANGED METHOD: PUBLIC STATIC java.lang.String generateRandomPassword()
	===  UNCHANGED METHOD: PUBLIC STATIC boolean getBooleanProperty(java.util.Properties, java.lang.String, boolean)
	===  UNCHANGED METHOD: PUBLIC STATIC java.lang.String getCanonicalFilePathProperty(java.util.Properties, java.lang.String, java.lang.String)
	===  UNCHANGED METHOD: PUBLIC STATIC int getIntegerProperty(java.util.Properties, java.lang.String, int)
	===  UNCHANGED METHOD: PUBLIC STATIC java.lang.String getRequiredProperty(java.util.Properties, java.lang.String)
		===  UNCHANGED EXCEPTION: java.util.NoSuchElementException
	+++  NEW METHOD: PUBLIC(+) STATIC(+) java.lang.String getRequiredProperty(java.util.Properties, java.lang.String, java.lang.String)
		+++  NEW EXCEPTION: java.util.NoSuchElementException
	===  UNCHANGED METHOD: PUBLIC STATIC java.lang.String getSection(java.lang.String, int)
		===  UNCHANGED EXCEPTION: java.lang.IllegalArgumentException
	===  UNCHANGED METHOD: PUBLIC STATIC java.lang.String getStringProperty(java.util.Properties, java.lang.String, java.lang.String)
	+++  NEW METHOD: PUBLIC(+) STATIC(+) java.lang.String getStringProperty(java.util.Properties, java.lang.String, java.lang.String, java.lang.String)
	===  UNCHANGED METHOD: PUBLIC STATIC boolean isNumber(java.lang.String)
	===  UNCHANGED METHOD: PUBLIC STATIC boolean isPositive(java.lang.String)
	===  UNCHANGED METHOD: PUBLIC STATIC java.lang.String native2Ascii(java.lang.String)
	===  UNCHANGED METHOD: PUBLIC STATIC java.lang.String normalizePostData(java.lang.String)
	===  UNCHANGED METHOD: PUBLIC STATIC int parseIntParameter(java.lang.String, int)
	===  UNCHANGED METHOD: PUBLIC STATIC java.lang.String repeatString(java.lang.String, int)
	===  UNCHANGED METHOD: PUBLIC STATIC java.lang.String replaceEntities(java.lang.String)
	===  UNCHANGED METHOD: PUBLIC STATIC java.lang.String replaceString(java.lang.String, java.lang.String, java.lang.String)
	===  UNCHANGED METHOD: PUBLIC STATIC java.lang.String replaceString(java.lang.String, int, int, java.lang.String)
	===  UNCHANGED METHOD: PUBLIC STATIC java.lang.String replaceStringCaseUnsensitive(java.lang.String, java.lang.String, java.lang.String)
	===  UNCHANGED METHOD: PUBLIC STATIC java.lang.String toHexString(byte[])
	===  UNCHANGED METHOD: PUBLIC STATIC java.lang.String urlDecode(java.lang.String, java.lang.String)
		===  UNCHANGED EXCEPTION: java.lang.IllegalArgumentException
	===  UNCHANGED METHOD: PUBLIC STATIC java.lang.String urlDecodeUTF8(java.lang.String)
	===  UNCHANGED METHOD: PUBLIC STATIC java.lang.String urlEncode(java.lang.String, java.lang.String)
	===  UNCHANGED METHOD: PUBLIC STATIC java.lang.String urlEncodeUTF8(java.lang.String)
===  UNCHANGED CLASS: PUBLIC org.apache.wiki.util.TimedCounterList  (not serializable)
	===  CLASS FILE FORMAT VERSION: 52.0 <- 52.0
	===  UNCHANGED INTERFACE: java.util.List
	===  UNCHANGED INTERFACE: java.util.Collection
	===  UNCHANGED INTERFACE: java.lang.Iterable
	===  UNCHANGED SUPERCLASS: java.util.AbstractList (<- java.util.AbstractList)
	===  UNCHANGED CONSTRUCTOR: PUBLIC TimedCounterList()
	===  UNCHANGED METHOD: PUBLIC void add(int, java.lang.Object)
	===  UNCHANGED METHOD: PUBLIC void cleanup(long)
	===  UNCHANGED METHOD: PUBLIC int count(java.lang.Object)
	===  UNCHANGED METHOD: PUBLIC java.lang.Object get(int)
	===  UNCHANGED METHOD: PUBLIC long getAddTime(int)
	===  UNCHANGED METHOD: PUBLIC java.lang.Object remove(int)
	===  UNCHANGED METHOD: PUBLIC java.lang.Object set(int, java.lang.Object)
	===  UNCHANGED METHOD: PUBLIC int size()
===  UNCHANGED ENUM: PUBLIC FINAL org.apache.wiki.util.XHTML  (compatible)
	===  CLASS FILE FORMAT VERSION: 52.0 <- 52.0
	===  UNCHANGED INTERFACE: java.lang.Comparable
	===  UNCHANGED INTERFACE: java.io.Serializable
	===  UNCHANGED SUPERCLASS: java.lang.Enum (<- java.lang.Enum)
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL org.apache.wiki.util.XHTML big
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL org.apache.wiki.util.XHTML select
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL org.apache.wiki.util.XHTML legend
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL org.apache.wiki.util.XHTML caption
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL org.apache.wiki.util.XHTML textarea
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL org.apache.wiki.util.XHTML del
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL org.apache.wiki.util.XHTML hr
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL java.lang.String XMLNS_xhtml
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL java.lang.String FRAMESET_DTD_PubId
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL org.apache.wiki.util.XHTML abbr
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL org.apache.wiki.util.XHTML frameset
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL org.apache.wiki.util.XHTML var
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL java.lang.String ATTR_value
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL java.lang.String ATTR_colspan
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL java.lang.String ATTR_src
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL org.apache.wiki.util.XHTML dfn
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL org.apache.wiki.util.XHTML script
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL java.lang.String ATTR_name
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL org.apache.wiki.util.XHTML input
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL org.apache.wiki.util.XHTML meta
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL org.apache.wiki.util.XHTML style
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL org.apache.wiki.util.XHTML object
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL org.apache.wiki.util.XHTML sub
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL org.apache.wiki.util.XHTML strong
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL org.apache.wiki.util.XHTML samp
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL java.lang.String ATTR_rows
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL org.apache.wiki.util.XHTML isindex
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL org.apache.wiki.util.XHTML sup
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL org.apache.wiki.util.XHTML br
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL java.lang.String ATTR_height
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL java.lang.String ATTR_align
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL java.lang.String ATTR_content
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL org.apache.wiki.util.XHTML iframe
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL org.apache.wiki.util.XHTML map
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL org.apache.wiki.util.XHTML table
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL java.lang.String ATTR_cellspacing
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL org.apache.wiki.util.XHTML a
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL org.apache.wiki.util.XHTML b
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL java.lang.String CSS_style
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL org.apache.wiki.util.XHTML address
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL org.apache.wiki.util.XHTML acronym
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL org.apache.wiki.util.XHTML strike
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL org.apache.wiki.util.XHTML i
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL org.apache.wiki.util.XHTML bdo
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL org.apache.wiki.util.XHTML menu
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL java.lang.String ATTR_width
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL org.apache.wiki.util.XHTML p
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL org.apache.wiki.util.XHTML td
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL org.apache.wiki.util.XHTML tfoot
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL org.apache.wiki.util.XHTML q
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL org.apache.wiki.util.XHTML s
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL org.apache.wiki.util.XHTML th
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL org.apache.wiki.util.XHTML u
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL org.apache.wiki.util.XHTML cite
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL org.apache.wiki.util.XHTML li
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL org.apache.wiki.util.XHTML tr
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL org.apache.wiki.util.XHTML font
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL org.apache.wiki.util.XHTML span
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL org.apache.wiki.util.XHTML dd
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL org.apache.wiki.util.XHTML small
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL org.apache.wiki.util.XHTML tt
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL java.lang.String ATTR_style
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL org.apache.wiki.util.XHTML col
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL java.lang.String ATTR_border
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL org.apache.wiki.util.XHTML optgroup
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL org.apache.wiki.util.XHTML tbody
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL org.apache.wiki.util.XHTML dl
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL org.apache.wiki.util.XHTML fieldset
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL org.apache.wiki.util.XHTML body
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL org.apache.wiki.util.XHTML dir
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL org.apache.wiki.util.XHTML div
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL org.apache.wiki.util.XHTML dt
	===  UNCHANGED FIELD: PUBLIC STATIC java.lang.String FRAMESET_DTD_SysId
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL org.apache.wiki.util.XHTML ul
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL org.apache.wiki.util.XHTML html
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL java.lang.String ATTR_href
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL org.apache.wiki.util.XHTML area
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL java.lang.String ATTR_checked
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL org.apache.wiki.util.XHTML blockquote
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL org.apache.wiki.util.XHTML pre
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL java.lang.String ATTR_httpEquiv
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL java.lang.String ATTR_scheme
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL java.lang.String ATTR_selected
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL java.lang.String ATTR_size
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL java.lang.String ATTR_cols
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL org.apache.wiki.util.XHTML em
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL org.apache.wiki.util.XHTML noframes
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL org.apache.wiki.util.XHTML noscript
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL java.lang.String ATTR_bgcolor
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL java.lang.String ATTR_id
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL org.apache.wiki.util.XHTML applet
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL org.apache.wiki.util.XHTML frame
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL org.apache.wiki.util.XHTML option
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL org.apache.wiki.util.XHTML img
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL org.apache.wiki.util.XHTML code
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL org.apache.wiki.util.XHTML thead
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL org.apache.wiki.util.XHTML link
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL org.apache.wiki.util.XHTML h1
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL org.apache.wiki.util.XHTML h2
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL java.lang.String ATTR_cellpadding
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL org.apache.wiki.util.XHTML h3
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL org.apache.wiki.util.XHTML basefont
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL org.apache.wiki.util.XHTML h4
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL org.apache.wiki.util.XHTML title
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL org.apache.wiki.util.XHTML h5
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL org.apache.wiki.util.XHTML h6
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL org.apache.wiki.util.XHTML button
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL org.apache.wiki.util.XHTML head
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL java.lang.String TRANSITIONAL_DTD_PubId
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL org.apache.wiki.util.XHTML param
	===  UNCHANGED FIELD: PUBLIC STATIC java.lang.String TRANSITIONAL_DTD_SysId
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL java.lang.String ATTR_class
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL org.apache.wiki.util.XHTML ol
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL java.lang.String ATTR_alt
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL java.lang.String ATTR_type
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL java.lang.String ATTR_title
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL org.apache.wiki.util.XHTML center
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL org.apache.wiki.util.XHTML colgroup
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL org.apache.wiki.util.XHTML label
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL org.apache.wiki.util.XHTML kbd
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL org.apache.wiki.util.XHTML ins
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL org.apache.wiki.util.XHTML form
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL java.lang.String ATTR_rel
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL java.lang.String STRICT_DTD_PubId
	===  UNCHANGED FIELD: PUBLIC STATIC java.lang.String STRICT_DTD_SysId
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL org.apache.wiki.util.XHTML base
	===  UNCHANGED METHOD: PUBLIC java.lang.String getName()
	===  UNCHANGED METHOD: PUBLIC STATIC java.lang.String getNamedCharacterEntity(int)
	===  UNCHANGED METHOD: PUBLIC STATIC org.apache.wiki.util.XHTML valueOf(java.lang.String)
	===  UNCHANGED METHOD: PUBLIC STATIC org.apache.wiki.util.XHTML[] values()
===  UNCHANGED CLASS: PUBLIC FINAL org.apache.wiki.util.XhtmlUtil  (not serializable)
	===  CLASS FILE FORMAT VERSION: 52.0 <- 52.0
	===  UNCHANGED SUPERCLASS: java.lang.Object (<- java.lang.Object)
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL org.jdom2.output.Format EXPAND_EMPTY_NODES
	===  UNCHANGED METHOD: PUBLIC STATIC org.jdom2.Element element(org.apache.wiki.util.XHTML)
	===  UNCHANGED METHOD: PUBLIC STATIC org.jdom2.Element element(org.apache.wiki.util.XHTML, java.lang.String)
	===  UNCHANGED METHOD: PUBLIC STATIC org.jdom2.Element img(java.lang.String, java.lang.String)
	===  UNCHANGED METHOD: PUBLIC STATIC org.jdom2.Element input(java.lang.String, java.lang.String, java.lang.String)
	===  UNCHANGED METHOD: PUBLIC STATIC org.jdom2.Element link(java.lang.String, java.lang.String)
	===  UNCHANGED METHOD: PUBLIC STATIC java.lang.String serialize(org.jdom2.Element)
	===  UNCHANGED METHOD: PUBLIC STATIC java.lang.String serialize(org.jdom2.Element, boolean)
	===  UNCHANGED METHOD: PUBLIC STATIC java.lang.String serialize(org.jdom2.Element, org.jdom2.output.Format)
	===  UNCHANGED METHOD: PUBLIC STATIC void setClass(org.jdom2.Element, java.lang.String)
	===  UNCHANGED METHOD: PUBLIC STATIC org.jdom2.Element target(java.lang.String, java.lang.String)
===  UNCHANGED CLASS: PUBLIC FINAL org.apache.wiki.util.XmlUtil  (not serializable)
	===  CLASS FILE FORMAT VERSION: 52.0 <- 52.0
	===  UNCHANGED SUPERCLASS: java.lang.Object (<- java.lang.Object)
	===  UNCHANGED METHOD: PUBLIC STATIC java.lang.String extractTextFromDocument(org.jdom2.Document)
	===  UNCHANGED METHOD: PUBLIC STATIC org.jdom2.Element getXPathElement(org.jdom2.Element, java.lang.String)
	===  UNCHANGED METHOD: PUBLIC STATIC java.util.List parse(java.lang.String, java.lang.String)
	===  UNCHANGED METHOD: PUBLIC STATIC java.util.List parse(java.io.InputStream, java.lang.String)
